src/Domain/Creators/TimeDependence/UniformRotationAboutZAxis.cpp

```diff
@@ -69,9 +68,9 @@ UniformRotationAboutZAxis<MeshDim>::functions_of_time() const noexcept {
   // We use a `PiecewisePolynomial` with 2 derivs since some transformations
   // between different frames for moving meshes can require Hessians.
   result[function_of_time_name_] =
-      std::make_unique<FunctionsOfTime::PiecewisePolynomial<2>>(
-          initial_time_,
-          std::array<DataVector, 3>{{{0.0}, {angular_velocity_}, {0.0}}});
+      std::make_unique<FunctionsOfTime::PiecewisePolynomial<3>>(
```

Comment:
I'm actually confused by the comment, which just got copied and pasted from UniformTranslation, which @nilsdeppe wrote. @nilsdeppe could you please explain what the concern behind your comment is, and whether I have anything to worry about changing this to a 3rd order piecewise polynomial, which is what I want to read in from spec?

Reply:
I updated the comment in my next fixup commit, after discussing with Nils Deppe. I will also move this to a separate commit, since the change is in its own file and that's easy to do